From 8d19fb5979522032cf4d96909d609aceac0dde85 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:13:30 +0800
Subject: [PATCH] MLK-11123-30 ARM: dts: disable otg features for
 imx7d-12x12-lpddr3-arm2 board

Disable all otg features for otg1 port of imx7d-12x12-lpddr3-arm2 board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index c50fbd1..d0f7480 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -440,11 +440,17 @@
 
 &usbotg1 {
 	vbus-supply = <&reg_usb_otg1_vbus>;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
 &usbotg2 {
 	vbus-supply = <&reg_usb_otg2_vbus>;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.8.0

